backend/controllers/roleValidation.ts

```diff
@@ -56,30 +56,43 @@ const validateRoleDescriptionFormat = (roleDescription) => {
   return errors
 }
 
-const validateRoleDescription = (roleDescription) => {
+const validateRoleDescription = (roleDescription: string) => {
   return validateRoleDescriptionFormat(roleDescription)
 }
 
-const validateCreateRole = ({ roleCode, roleName, roleDescription, adminType }) => {
+const validateRoleAdminType = (adminType: string[]) => {
+  const errors = []
+  if (!adminType?.length) {
+    errors.push({ href: '#adminType', text: 'Select an admin type' })
+  }
+  return errors
+}
+
+const validateCreateRole = (obj: {
```

Comment:
`obj` isn't a great name...

Reply:
yes -  I will update it - auto update from js to ts that I missed